src/settings-ui/Settings.UI/Views/PowerLauncherPage.xaml

```diff
@@ -246,6 +289,14 @@
                                                                  MinWidth="{StaticResource SettingActionControlMinWidth}" />
                                                     </controls:Setting.ActionContent>
                                                 </controls:Setting>
+
+                                                <controls:Setting x:Uid="PowerLauncher_PluginWeightBoost" Style="{StaticResource ExpanderContentSettingStyle}" IsEnabled="{x:Bind Enabled, Mode=OneWay}">
```

Comment:
We should move that after global results checkbox (because of infobars and logic) and it should be greyed out if plugin is not shown in global results.